lib/Target/LLVMIR/LLVMDIScope.cpp

```diff
@@ -74,16 +74,29 @@ struct LLVMDIScopePass : public LLVMDIScopeBase<LLVMDIScopePass> {
           context, llvm::sys::path::filename(inputFilePath),
           llvm::sys::path::parent_path(inputFilePath));
     }
-    auto distinctId = mlir::DistinctAttr::create(mlir::UnitAttr::get(context));
-    if (!compileUnitAttr) {
-      compileUnitAttr = LLVM::DICompileUnitAttr::get(
-          context, distinctId, llvm::dwarf::DW_LANG_C, fileAttr,
-          StringAttr::get(context, "triton"),
-          /*isOptimized=*/true, LLVM::DIEmissionKind::LineTablesOnly);
-    }
     auto subroutineTypeAttr =
         LLVM::DISubroutineTypeAttr::get(context, llvm::dwarf::DW_CC_normal, {});
 
+    // Figure out debug information (`subprogramFlags` and `compileUnitAttr`) to
+    // attach to the function definition / declaration. External functions are
+    // declarations only, and are defined in a different compile unti, so mark
```

Comment:
s/uni/unit/

Reply:
Thanks! Already had it fixed locally, but forgot `git add`.